.scripts/regeneration.js

```diff
@@ -53,22 +53,22 @@ const goMappings = {
 for (namespace in goMappings) {
     // for each swagger run the autorest command to generate code based on the swagger for the relevant namespace and output to the /generated directory
     const inputFile = swaggerDir + goMappings[namespace];
-    generate(inputFile, 'test/autorest/' + namespace, '--head-as-boolean=true');
+    generate(inputFile, 'test/autorest/' + namespace);
 }
 
-const blobStorage = 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/storage-dataplane-preview/specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-07-07/blob.json';
+const blobStorage = 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1a7b2bbb84370bd96c0bf53f13f56f4fc2cafab9/specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-07-07/blob.json';
 generate(blobStorage, 'test/storage/2019-07-07/azblob', '--credential-scope="https://storage.azure.com/.default" --module="azstorage" --openapi-type="data-plane"');
 
-const network = 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/network/resource-manager/readme.md';
-generateFromReadme(network, 'package-2020-03', 'test/network/2020-03-01/armnetwork', '--credential-scope="https://management.azure.com//.default" --armcore-connection=true --module=armnetwork');
+const network = 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1c100daabed6a008c76f3e4e0cae4e7eac5a5bf4/specification/network/resource-manager/readme.md';
+generateFromReadme(network, 'package-2020-03', 'test/network/2020-03-01/armnetwork', '--credential-scope="https://management.azure.com//.default" --module=armnetwork --azure-arm=true');
```

Comment:
I understand this is to avoid CI failures, just wondering if this could lead us to not catch bugs in new features for generation as they come out? (Since we added these to test our generation against actual swaggers.) Not saying we absolutely shouldnt do this...just wondering if this will hide things we should be aware of?

Reply:
It's a trade-off of stability vs. being on the bleeding edge.  For reference, we do the same thing with the test server to avoid this same issue.  It does mean that we will need to update these periodically though (same with the test server).  Another benefit is that since the inputs are stable, it reduces the noise in the diff due to codegen changes.

Reply:
Ok it's fine. I think overall if we update every so often it shouldnt be a huge problem.